decidim-admin/app/commands/decidim/admin/create_attachment_collection.rb

```diff
@@ -32,12 +33,20 @@ def call
       attr_reader :form
 
       def create_attachment_collection
-        AttachmentCollection.create!(
+        Decidim.traceability.create!(
+          AttachmentCollection,
+          @user,
+          params
+        )
+      end
+
+      def params
```

Comment:
Let's change this to attributes to be more consistent with others commands 
```suggestion
      def attributes
```